test/circuit/library/ansatzes/test_ucc.py

```diff
@@ -1,6 +1,6 @@
 # This code is part of Qiskit.
 #
-# (C) Copyright IBM 2021.
+# (C) Copyright IBM 2021, 2022.
```

Comment:
All these files that were changed before and are now in essence unchanged can you put the date back to 2021 so they get dropped completely from the PR and we are not simply pushing some date change on these